### ISSUE TITLE
Multiple messengers (Mattermost and Slack) at the same time?

### ISSUE THREAD

**Issue by bor8** [alphanodes/redmine_messenger#77]
Hello, is it possible to serve several messengers at the same time from the same project? I.e. one status message appears in Slack and in Mattermost at the same time? If not, I submit this as a feature request ;)

**Comment by alexandermeindl** [comment]
No, this is not possible and there are no plans to add this feature.

**Comment by bor8** [comment]
Thanks. Well, in https://github.com/altsol/redmine_mattermost it is possible via custom fields. Maybe - just maybe - someone can take the code from there. ;)
